eng/targets/Imports.targets

```diff
@@ -368,4 +368,12 @@
       <Reference Remove="@(Reference)" Condition=" '%(FileName)' == 'Microsoft.VisualBasic' or '%(FileName)' == 'Microsoft.VisualBasic.Core' " />
     </ItemGroup>
   </Target>
+
+  <Target Name="WorkaroundDotnetMsbuild7161"
```

Comment:
Small request: stick a comment with the explicit bug URL.

Reply:
➡️ Done